NitroxPatcher/Patches/Dynamic/Creature_ChooseBestAction_Patch.cs

```diff
@@ -16,7 +15,10 @@ public class Creature_ChooseBestAction_Patch : NitroxPatch, IDynamicPatch
 
         public static bool Prefix(Creature __instance, ref CreatureAction __result)
         {
-            NitroxId id = NitroxEntity.GetId(__instance.gameObject);
+            if (!__instance.TryGetIdOrWarn(out NitroxId id))
```

Comment:
This will probably be very noisy if it happens.  Do we have a way to combat this?

Reply:
Like mentioned above this patch is not used. In theory we could use `Log.WarnOnce` which would only warn once per id.

Reply:
This sounds good, let's make it so this method can only ever log once per id.